index.html

```diff
@@ -57,7 +57,7 @@ <h1>
         Last Modified:
       </dt>
       <dd>
-        1 January 2020
+        7 February 2020
       </dd>
     </dl>
     <h2 id="goals">
```

Comment:
The goals section should include Window Segments in the incubations list with Screen Enumeration and Window Placement.

Reply:
Good catch @michaelwasserman! I pushed a fix.